src/base.config.ts

```diff
@@ -401,6 +403,10 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 							has: 'web-animations'
 						}
 					]
+				}),
+			watch &&
+				new ExtraWatchWebpackPlugin({
+					files: ['!(output|.*)/**']
```

Comment:
does this ignore node_modules by default?

Reply:
That was the behavior I observed in testing, but it's not explicitly documented as such so I've added it here.